src/pretix/static/pretixcontrol/js/ui/hidequota.js

```diff
@@ -0,0 +1,30 @@
+$(document).ready(function() {
+  hideDeselected();
+});
+
+function hideDeselected() {
+  var v = $("#id_quota_option").val();
+
```

Comment:
I like your proposal here for a soft animation instead of a `hide()`. Unfortunately, it seems as though `slideToggle` doesn't currently support a boolean condition you can pass in like that. So I still think we'd need `hideAll`, to initialize the fields to be hidden when coming to the page.